docs/miscellaneous/update-kb.md

```diff
@@ -1,25 +1,28 @@
 ---
-title: 'Updating the Knowledgebase'
+title: 'Updating the Knowledge Base DNS'
 sidebar_position: 3
 ---
 
-The goal of this Knowledgebase is to provide everyone with the most up-to-date information on all kinds of AdGuard DNS-related topics. But things constantly change, and sometimes an article doesn't reflect the current state of things anymore — there are simply not so many of us to keep an eye on every single bit of information and update it accordingly when new versions are released. 
+The goal of this Knowledge Base is to provide everyone with the most up-to-date information on all kinds of AdGuard DNS-related topics. But things constantly change, and sometimes an article doesn't reflect the current state of things anymore — there are simply not so many of us to keep an eye on every single bit of information and update it accordingly when new versions are released. 
 
 This is why we placed all of our KB content to [GitHub](https://github.com/AdguardTeam/KnowledgeBaseDNS), and now literally anyone can contribute to it by suggesting edits and translations to existing articles, as well as totally new ones. 
 
-## How to suggest a change or write a new article
+## How to suggest a change or write a new article 
  
-You can suggest changes to current articles and add new ones to the Knowledgebase VPN using the functionality of the GitHub mentioned above.
-If you are unfamiliar with principles of working with the platform, start by reading [articles in this section](https://docs.github.com/en).
- 
-Once you are ready to start, work in [the KnowledgeBaseDNS repository](https://github.com/AdguardTeam/KnowledgeBaseDNS). Alter articles using [Markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) in any source-code editor you like. 
+You can suggest changes to current articles and add new ones to the Knowledge Base DNS using the functionality of the GitHub mentioned above.
+If you are unfamiliar with principles of working with the platform, start by reading [documentation in this section](https://docs.github.com/en).
+
+The Knowledge Base DNS website is buit using Docusaurus 2 — a modern static website generator. When suggesting changes or additions, take into account that all documents must comply with the principles of the platform. You can read about them in [the guide available via this link](https://docusaurus.io/docs/category/guides).
+
+You can use Docusaurus functionality to build a site from documents placed locally on your computer. Detailed instructions on how to do this can be found [here](https://github.com/AdguardTeam/KnowledgeBaseDNS/blob/main/README.md). 
```

Comment:
You can deploy this Knowledge Base locally on your computer to preview the changes you suggest. Detailed instructions on how to do this can be found [in the README.md file](https://github.com/AdguardTeam/KnowledgeBaseDNS/blob/main/README.md) on this Knowledge Base's GitHub page.